app/views/application/_checklist_execution_results.html.erb

```diff
@@ -24,7 +24,7 @@
     <tr>
       <td><%= link_to 'View Results', new_task_test_execution_path(execution.task) %></td>
       <% # for both c1 and c3 executions (if they exist) %>
-      <% [execution, c3_execution].reject { |exec| exec == false }.each do |exec| %>
+      <% [execution, c3_execution].reject { |exec| exec.nil? }.each do |exec| %>
```

Comment:
cool, this change makes sense. I didn't realize the `sibling_execution()` function on a `test_execution` returned `nil` and not `false`